packages/backend/src/managers/recipes/PodManager.ts

```diff
@@ -86,6 +84,7 @@ export class PodManager implements Disposable {
 
     return pods.find(pod => {
       const labels = pod.Labels;
+      // eslint-disable-next-line sonarjs/different-types-comparison
```

Comment:
from my understanding, labels is never undefined (can be empty) or it means the API is lying 
but then the if should be removed if empty and we trust the contract

Reply:
Seems an issue with the API as I was able to create a pod (mypod) and podman pod inspect mypod does not return the Labels field